packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -465,7 +465,19 @@ class SlideshowView extends GalleryComponent {
 
     if (nextKeys.includes(code)) {
       e.preventDefault();
-      this._next({ direction: getDirection(code), isKeyboardNavigation: true });
+      const activeItemIdx = window.document.activeElement.getAttribute('data-idx');
+
+      const shouldOutOfGallery =
```

Comment:
I think the name of the const should be changed,
check code first,
what if there is no totalItemsCount?

Reply:
if is no totalItemsCount, some of our logic will not work properly because also use it in other places,
So I think our consumers must always pass it on,